lib/js/components/DatePickers/DatePicker/DatePicker.vue

```diff
@@ -279,15 +284,10 @@ export default defineComponent({
 		} = initFlatpickr({
 			props,
 			onChange,
+			onClose,
 			defaultDates: props.date ?? new Date(),
 			mode: 'single',
 		});
-		watch([() => props.isInteractive, () => props.state], async () => {
```

Comment:
Removing this watcher created a weird scenario. 
1. Open a date picker
2. Set `isInteractive` prop to false
3. You can still select a date. I don't know if this is a problem, tho. Let it be for now.